src/std/misc/string.ss

```diff
@@ -172,3 +174,17 @@ package: std/misc
        (old-empty? (subst-helper-empty-old str new count))
        (str-empty? str)
        (else       (subst-helper-nonempty-old str old new count))))))
+
+;; Returns true when the string s consists only of whitespace characters.
+;;
+;;   " "   space
+;;   "\n"  line feed
+;;   "\t"  horizontal tab
+;;   "\r"  carriage return
+;;   "\f"  form feed
+;;   "\v"  vertical tab
+;;
+;; Example:
+;;  (string-whitespace? " \n\r \t") => #t
+(def (string-whitespace? s)
+  (char-set-every char-whitespace? (->char-set s)))
```

Comment:
Just curious, what are `char-set`s handling that srfi13's `string-every` isn't?

```scheme
(string-every char-whitespace? " \n")
#t
```

Reply:
I think that this is better written with `string-every`, as @dwdv suggests.

Reply:
Thanks for your feedback.